packages/containers/src/ComponentForm/ComponentForm.md

```diff
@@ -2,39 +2,96 @@
 
 This component has been designed to take URLs of the service has entry point and display the corresponding form
 
+## Concept
+
+`uiSpec` is a datastructure that represent the form itself. It is an object composed of three attributes:
+
+- `jsonSchema` the schema that let the backend describe the data structure to submit
+- `uiSchema` the schema that let the frontend build the form
+- `properties` the original payload (if you want to edit a data)
+
+The backend used with this component is described at https://talend.github.io/component-runtime.
+
 ## How to use
 
 The common use case is to edit a component:
 
 ```javascript
 import ComponentForm from '@talend/react-containers/lib/ComponentForm';
 export default function MyComponent(props) {
-    return (
-        <ComponentForm
-            definitionURL="/api/v1/components/component-id"
-            triggerURL="/api/v1/action/execute"
-            submitURL="/api/v1/persist"
-        />
-    );
+	return (
+		<ComponentForm
+			definitionURL="/api/v1/components/component-id"
+			triggerURL="/api/v1/action/execute"
+			submitURL="/api/v1/persist"
+		/>
+	);
 }
 
 // in a saga
 function* handleForm() {
-    const action = yield take(ComponentForm.ON_SUBMIT_SUCCEED);
-    // do what you want with it
-    action === {
-        type: ComponentForm.ON_SUBMIT_SUCCEED,
-        data,
-        componentId
-    }
+	const action = yield take(ComponentForm.ON_SUBMIT_SUCCEED);
+	// do what you want with it
+	action ===
+		{
+			type: ComponentForm.ON_SUBMIT_SUCCEED,
+			data,
+			componentId,
+		};
 }
 ```
 
-definitionURL: url to fetch the form uiSpec
-triggerURL: url to fetch on trigger
-submitURL: url to POST the content
+## Props
+
+| name             | type   | desc                                                     |
+| ---------------- | ------ | -------------------------------------------------------- |
+| definitionURL\*  | string | url to GET the `uiSpec`                                  |
+| triggerURL\*     | string | url to POST on event trigger                             |
+| [submitURL]      | string | url to POST the content if action is of type "submit"    |
+| [uiSpecPath]     | string | to get the `uiSpec` from the result of GET definitionURL |
+| [lang]           | string | language code                                            |
```

Comment:
need more explanation, for example it is used to handle translation check (link to doc)